scripts/snowflake_generate_grant_all/main.go

```diff
@@ -0,0 +1,202 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"sort"
+	"strings"
+
+	"github.com/chanzuckerberg/terraform-provider-snowflake/pkg/provider"
+	"github.com/chanzuckerberg/terraform-provider-snowflake/pkg/resources"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/errgo.v2/fmt/errors"
+)
+
+const (
+	topLvlComment string = `
+	NOTE: Generated by scripts/snowflake_generate_grant_all.
+	      Changes made directly to this file will be overwritten.
```

Comment:
```suggestion
	topLvlComment string = `NOTE: Generated by scripts/snowflake_generate_grant_all.
	      Changes made directly to this file will be overwritten.
```

remove leading whitespace